core/src/test/scala/cromwell/util/SampleWdl.scala

```diff
@@ -1813,512 +1273,6 @@ object SampleWdl {
-        |  command {
-        |    cat non_existent_file
-        |  }
-        |  runtime {
-        |    continueOnReturnCode: true
-        |  }
-        |}
-        |task echoWithFalseContinueOnReturnCode {
-        |  command {
-        |    cat non_existent_file
-        |  }
-        |  runtime {
-        |    continueOnReturnCode: false
-        |  }
-        |}
-        |task echoWithTrueStringContinueOnReturnCode {
-        |  command {
-        |    cat non_existent_file
-        |  }
-        |  runtime {
-        |    continueOnReturnCode: "true"
-        |  }
-        |}
-        |task echoWithFalseStringContinueOnReturnCode {
-        |  command {
-        |    cat non_existent_file
-        |  }
-        |  runtime {
-        |    continueOnReturnCode: "false"
-        |  }
-        |}
-        |
-        |workflow echo_wf {
-        |  call echoWithSingleContinueOnReturnCode
-        |  call echoWithExpressionContinueOnReturnCode
-        |  call echoWithListContinueOnReturnCode
-        |  call echoWithTrueContinueOnReturnCode
-        |  call echoWithFalseContinueOnReturnCode
-        |  call echoWithTrueStringContinueOnReturnCode
-        |  call echoWithFalseStringContinueOnReturnCode
-        |}
-      """.stripMargin
-
-    val rawInputs = Map.empty[String, Any]
-  }
-
-  object WorkflowWithFullGooglyConfig extends SampleWdl {
-    override def wdlSource(runtime: String = "") =
-      """
-        |task googly_task {
-        |  command {
-        |    echo "Hello JES!"
-        |  }
-        |  runtime {
-        |    docker: "ubuntu:latest"
-        |    memory: "4G"
-        |    cpu: "3"
-        |    zones: "US_Metro US_Backwater"
-        |    disks: "/mnt/some-dir 3 SSD, /tmp/mnt 500 HDD"
-        |  }
-        |}
-        |
-        |workflow googly_workflow {
-        |  call googly_task
-        |}
-      """.stripMargin
-
-    val rawInputs = Map.empty[String, Any]
-  }
-
-  object WorkflowWithoutGooglyConfig extends SampleWdl {
-    override def wdlSource(runtime: String = "") =
-      """
-        |task googly_task {
-        |  command {
-        |    echo "Hello JES!"
-        |  }
-        |  runtime {
-        |    docker: "ubuntu:latest"
-        |  }
-        |}
-        |
-        |workflow googly_workflow {
-        |  call googly_task
-        |}
-      """.stripMargin
-
-    val rawInputs = Map.empty[String, Any]
-  }
-
-  object WorkflowWithLocalDiskGooglyConfig extends SampleWdl {
-    override def wdlSource(runtime: String = "") =
-      """
-        |task googly_task {
-        |  command {
-        |    echo "Hello JES!"
-        |  }
-        |  runtime {
-        |    docker: "ubuntu:latest"
-        |    disks: "local-disk 123 HDD"
-        |  }
-        |}
-        |
-        |workflow googly_workflow {
-        |  call googly_task
-        |}
-      """.stripMargin
-
-    val rawInputs = Map.empty[String, Any]
-  }
-
-  object WorkflowWithMessedUpMemory extends SampleWdl {
```

Comment:
I dont think we have all these cases of MessedUpMemory in Centaur---and neither do I feel there's an emergency for it. Especially since its the error's that would be valuable, and that's going to be a loooot of manual work.